test/TestApps/PushNotificationsDemoApp/main.cpp

```diff
@@ -147,15 +143,15 @@ int main()
         std::cin.ignore();
     }
 
-    if (PushNotificationManager::IsActivatorSupported(PushNotificationRegistrationActivators::ComActivator))
-    {
-        // Don't unregister PushTrigger because we still want to receive push notifications from background infrastructure.
-        PushNotificationManager::UnregisterActivator(PushNotificationRegistrationActivators::ComActivator);
-    }
+    PushNotificationManager::Default().Unregister();
 
+    AppNotificationManager::Default().Unregister();
+
+    AppNotificationManager::Default().Unregister();
```

Comment:
I don't think we need to unregister it twice

Reply:
Maybe they meant UnregisterAll in the second call? But yeah, a second call isn't needed.

Reply:
Nah. That would hose notifications completely. I'm sure that wasn't the intention :) @pmpurifoy

Reply:
More likely a typo here. This actually would end up throwing the second time around.